website/blog/2023-10-11-silver.md

```diff
@@ -0,0 +1,474 @@
+---
+author: Katarzyna Marek
+title: Metals v1.1.0 - Silver
+authorImageURL: https://github.com/kasiaMarek.png
+---
+
+We're happy to announce the release of Metals v1.1.0, which brings stability fixes and takes another step towards bettering error detection and error reporting.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">101</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">92</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/60?closed=1]
+(https://github.com/scalameta/metals/milestone/60?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Support new Scala versions](#support-new-Scala-versions)
+- [Build server status](#build-server-status)
+- [Enable run code lens for Mill build server](#enable-run-code-lens-for-Mill-build-server)
+- [Convert single line comments to multiline](#convert-single-line-comments-to-multiline)
+- [Detect project root](#detect-project-root)
+- [Error reports in metals doctor](#error-reports-in-metals-doctor)
+- [Changes for clients](#changes-for-clients)
+- [Show bloop errors](#show-bloop-errors)
+
+## Support new Scala versions
+#### Support for Scala 3.3.1
+
+Among other things Scala 3.3.1 adds support for JDK 16 records in Java sources.
+
+For more information about the release take a look at
+https://github.com/lampepfl/dotty/releases/tag/3.3.1
+
+#### Support for Scala 2.13.12
+
+Among other things Scala 2.13.12 brings actionable diagnostics - compiler suggested fixes for some errors and warnings. Actionable diagnostics are available in metals as code actions.
+
+For more information about the release take a look at
+https://github.com/scala/scala/releases/tag/v2.13.12
+
+## Build server connection status
+
+Since the previous release metals detects and reports when the build server stops responding. This feature got stabilized and the information about the build server connection is now visible in the status bar. For clients that don't support this status bar yet, the error message will be displayed as before.
```

Comment:
```suggestion
Since the previous release metals detects and reports when the build server stops responding. Over the last months this feature was stabilized and the information about the build server connection is now visible in the status bar. For clients that don't support this status bar yet, the error message will be displayed as before.
```